### PR TITLE
add more attribute in exploration

### PR DESCRIPTION
**Pull request by yilinxu**

### CODE REVIEW

File: aids.niaiddata.org/portal/gitops.json [hunk 1]
```diff
@@ -228,14 +266,32 @@
...
         { "field": "bshcvstat", "name": "Baseline HCV Sero-status"},
         { "field": "cd4nadir", "name": "CD4 Nadir Prior to HAART"},
         { "field": "status", "name": "Summarized HIV Sero-status"},
         {"field": "project_id", "name": "Project ID"},
-        { "field": "hiv_status", "name": "HIV Status"}
+        {"field": "frstcncrd", "name": "First confirmed cancer year"},
+        {"field": "frstdmd", "name": "First visit year with diabetes"},
+        {"field": "frstdmmd", "name": "First visit year with all necessary components to determine diabetes"},
+        {"field": "frsthtnd", "name": "First visit year with hypertension"},
+        {"field": "frsthtnmd", "name": "First visit year with all necessary components to determine hypertension"},
```
Comment (mfshao) [inline aids.niaiddata.org/portal/gitops.json]:
```suggestion
        {"field": "frsthtnmd", "name": "First visit year with all necessary components to determine hypertension"}
```

File: aids.niaiddata.org/portal/gitops.json [hunk 2]
```diff
@@ -174,17 +174,55 @@
...
+          ]
+        }, {
+          "title": "Treatment",
+          "fields": [
+            "arthxbase",
+            "ehaart",
+            "frstartd",
+            "frsthaad",
+            "lastnoad",
+            "lastnohd",
```
Comment (mfshao) [inline aids.niaiddata.org/portal/gitops.json]:
```suggestion
            "lastnohd"
```

File: aids.niaiddata.org/etlMapping.yaml [hunk 3]
```diff
@@ -25,14 +25,43 @@ mappings:
...
       - path: comorbidities
         props:
           - name: bshbvstat
           - name: bshcvstat
+          - name: frstcncrd
+          - name: frstdmd
+          - name: frstdmmd
+          - name: frsthtnd
+          - name: frsthtnmd
+      - path: diagnoses
```
Comment (mfshao) [inline aids.niaiddata.org/etlMapping.yaml]: ETL is not happy with this part:
`the properties '[{'name': 'primary_diagnosis'}, {'name': 'comorbidity'}, {'name': 'comorbidity_anemia'}, {'name': 'comorbidity_diabetes'}, {'name': 'comorbidity_hepatic_disease'}, {'name': 'comorbidity_hiv'}, {'name': 'comorbidity_renal_disease'}, {'name': 'prior_treatment'}, {'name': 'year_of_diagnosis'}]' for 'diagnoses'
for parent 'diagnosis'
has multiplicity 'many_to_one' that cannot be used on in 'flatten_props'`

so we need to put everthing here into `aggregated_props`

Comment (abgeorge7) [reply aids.niaiddata.org/etlMapping.yaml]: you can sort it to grab the most recent one

Comment (abgeorge7) [reply aids.niaiddata.org/etlMapping.yaml]: like this for example: https://github.com/uc-cdis/cdis-manifest/blob/master/gen3.datastage.io/etlMapping.yaml#L35

**Pull request closed without merging**